src/data.py

```diff
@@ -241,24 +248,95 @@ def parse_static_op_hours(hours_list, eatery_id, dining_items):
+    end_time = end.strftime('%I:%M') + end_time[-2:].upper()
 
-  start_hour = start_hour.rjust(2, "0")
-  end_hour = end_hour.rjust(2, "0")
   end_date = start_date
-
-  if (int(end_hour) < int(start_hour) or end_hour == '12') and end_minute.endswith('am'):
+  if overnight or (end.strftime('%p') == 'AM' and end < start):
     year, month, day = start_date.split('-')
     next_day = date(int(year), int(month), int(day)) + timedelta(days=1)
     end_date = next_day.isoformat()
 
-  new_start = "{}:{}:{}".format(start_date, start_hour, start_minute)
-  new_end = "{}:{}:{}".format(end_date, end_hour, end_minute)
+  new_start = "{}:{}".format(start_date, start_time)
+  new_end = "{}:{}".format(end_date, end_time)
 
   return [new_start, new_end]
 
 def get_trillium_menu():
-  statics_json = get(STATIC_MENUS_URL).json()
+  statics_json = requests.get(STATIC_MENUS_URL).json()
   return parse_dining_items(statics_json['Trillium'][0])
+
+def parse_collegetown_eateries(collegetown_data):
+  for eatery in collegetown_data:
+    new_id = resolve_id(eatery, collegetown=True)
+    new_eatery = CollegetownEateryType(
+        address=eatery['location']['address1'],
+        categories=[cuisine['title'] for cuisine in eatery.get('categories', [])],
+        coordinates=parse_coordinates(eatery),
+        eatery_type='Collegetown Restaurant',
+        id=new_id,
+        image_url=eatery.get('image_url', ''),
+        name=eatery.get('name', ''),
+        operating_hours=parse_collegetown_hours(eatery),
+        payment_methods=PaymentMethodsType(
+            brbs=False,
+            cash=True,
+            cornell_card=False,
+            credit=True,
+            swipes=False,
+            mobile=False,
+        ),
+        phone=eatery.get('phone', 'N/A'),
+        price=eatery.get('price', ''),
+        url=eatery.get('url', ''),
+
+    )
+    collegetown_eateries[new_eatery.id] = new_eatery
+
+def parse_collegetown_hours(eatery):
+  hours_list = eatery.get('hours', [{}])[0].get('open', [])
+  # gets open hours from first dictionary in hours, empty dict-list provided to mimic hours format
+  new_operating_hours = []
+
+  for i in range(NUM_DAYS_STORED_IN_DB):
+    new_date = today + timedelta(days=i)
+    new_events = [event for event in hours_list if event['day'] == new_date.weekday()]
+    new_operating_hours.append(
+        CollegetownHoursType(
+            date=new_date.isoformat(),
+            events=parse_collegetown_events(new_events, new_date.isoformat())
+        )
+    )
+  return new_operating_hours
+
+def parse_collegetown_events(event_list, event_date):
+  new_events = []
+  for event in event_list:
+    start, end = format_time(
+        event.get('start', ''),
+        event.get('end', ''),
+        event_date,
+        hr24=True,
+        overnight=event.get('is_overnight', False)
+    )
+    new_event = CollegetownEventType(
+        description='General',
+        end_time=end,
+        start_time=start,
```

Comment:
drop trailing comma

Reply:
trailing commas are fine – also they're mandatory in JS so we might as well keep the convention